### PR TITLE
Update Go to v1.14.13 and linux deps

### LINKED ISSUE #19: Fails to build for linux mobile
It looks like we added libegl1 but it should have been libgles2 (or maybe we need both)?

Steps to replicate:

```
$ fyne-cross linux -arch arm64 -tags mobile
...
# github.com/fyne-io/mobile/gl
In file included from /go/pkg/mod/github.com/fyne-io/mobile@v0.1.2-0.20201127155338-06aeb98410cc/gl/work.go:32:
./work.h:12:10: fatal error: GLES3/gl3.h: No such file or directory
 #include <GLES3/gl3.h> // install on Ubuntu with: sudo apt-get install libegl1-mesa-dev libgles2-mesa-dev libx11-dev
          ^~~~~~~~~~~~~
```

### PR DESCRIPTION
**Pull request by lucor**
This PR is an attempt to update Go to latest versions.
Additionally fixes #19 

To test:
- ensure to have the latest version from develop: `go get github.com/fyne-io/fyne-cross@develop`
- build the docker images locally: `make build-images`
- clear the go cache: `go cache -clean -modcache`
- build your application with fyne-cross

### CODE REVIEW

**Comment by Jacalz** [comment]
Looks like the correct third command is `go clean -modcache`. Building the images locally to hopefully test this soon.

**Comment by Jacalz** [comment]
This works well from my tests. I could start the application on my mac running High Sierra and I also verified the Go version by including a piece of code that was introduced with Go 1.14.

**Review by Jacalz — DISMISSED** [review]

**Review by Jacalz — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-## Unreleased
+## [1.0.0]
 - Add support for "fyne release" #3
 - Add support for creating packaged .tar.gz bundles on freebsd #6
 - Add support for Linux Wayland #10
-- Update fyne cli to v1.4.2-0.20201204171445-8f33697cf611 (fyne-io#1538 fyne-io#1527)
+- Update fyne cli to v1.4.2 (fyne-io#1538 fyne-io#1527)
 - Deprecate `output` flag in favour of `name`
 - Fix env flag validation #14
+- Fix fails to build for linux mobile #19
```
Comment (Jacalz) [inline CHANGELOG.md:10]: Could this perhaps be “Fix build failure for Linux mobile” instead?

File: docker/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,14 @@ All notable changes to the fyne-cross docker images will be documented in this f
 Release cycle won't follow the fyne-cross one, so the images will be tagged using the label
 year.month.day along with the latest one.
 
+# Release 20.12.13
+- Update Go to v1.14.13
+- Fix fails to build for linux mobile #19
```
Comment (Jacalz) [inline docker/CHANGELOG.md:10]: Could this perhaps be “Fix build failure for Linux mobile” instead?

**Review by Jacalz — CHANGES_REQUESTED** [review]
Sorry for the requested changes right after approval. Spotted a few grammatical improvements that could be made.

File: CHANGELOG.md (hunk 1 above)
Comment (lucor) [reply CHANGELOG.md]: Thanks for the catch, fixed in b36aee34dc25a7404bef1d0edd6c61efceb257fe

File: docker/CHANGELOG.md (hunk 2 above)
Comment (lucor) [reply docker/CHANGELOG.md]: Thanks for the catch, fixed in b36aee34dc25a7404bef1d0edd6c61efceb257fe

**Review by Jacalz — APPROVED** [review]
Looks great 👍

**Review by andydotxyz — APPROVED** [review]
Seems to be working well thanks

**Pull request merged**